### PR TITLE
Feature/incremental updates

### PR DESCRIPTION
**Pull request by scholz-timo**

### CODE REVIEW

**Comment by scholz-timo** [comment]
Maybe we can somehow speed up the testing process, its currently taking more time since we download docker & docker-compose.

File: steps/helm/src/VersionStrategy/CurrentVersionMangager/implementation/NPMPackageProvider.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        next();
+      });
+
+      read.on('finish', () => reject(new Error('File not found')));
+
+      const readable = Readable.from(tarBuffer);
+      readable.pipe(read);
+    });
+
+    // const file = unzippers.files.find((file) => file.path.endsWith('versions.json'));
```
Comment (FCQCarryOut) [inline steps/helm/src/VersionStrategy/CurrentVersionMangager/implementation/NPMPackageProvider.ts:56]: code comment

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -9,11 +9,35 @@ jobs:
   test:
     runs-on: ubuntu-latest
     timeout-minutes: 5
+    #services:
+    #  verdaccio:
+    #    image: verdaccio/verdaccio
+    #    volumes: 
+    #      # - /steps/docker/verdaccio/config.yaml:/verdaccio/conf/config.yaml
+    #    ports:
+    #      - 4873:4873
```
Comment (FCQCarryOut) [inline .github/workflows/testing.yml:18]: cleanup

Comment (FCQCarryOut) [reply .github/workflows/testing.yml]: done

**Review by FCQCarryOut — APPROVED** [review]

**Pull request merged**